server/cartHandler.js

```diff
@@ -0,0 +1,75 @@
+var models = require("./models");
+var commons = require("./commons");
+var validator = require("./tokenValidator");
+var sequelize = require("sequelize");
+
+function addToCart(response, postData, authToken) {
+	if (postData == "") {
+		commons.badRequest("ERROR: no content", response);
+	} else {
+		console.log(postData);
+		
+		if (!validator.validateToken(response, authToken)) {
+			return;
+		}
+		
+		var cartJson = commons.getJson(postData, response, commons.badRequest);
+		if (cartJson == null) {
+			commons.internalServerError("ERROR: File is empty", response);
+			return;
+		}
+		
+		//Getting user login from authorization token
+		var objectString = authToken.slice(7);

```

Comment:
Avoid using 'magic' numbers (7). If you depend on some string (in this case 'Bearer') you should use Bearer.strlen or something.